cluster-up/cluster/kind/common.sh

```diff
@@ -88,13 +88,11 @@ function kind_up() {
     done
 
     $KIND --loglevel debug create cluster --retain --name=${CLUSTER_NAME} --config=${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/kind.yaml --image=$KIND_NODE_IMAGE
-    cp $($KIND get kubeconfig-path --name=${CLUSTER_NAME}) ${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/.kubeconfig
+    $KIND get kubeconfig --name=${CLUSTER_NAME} > ${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/.kubeconfig
 
     docker cp ${CLUSTER_NAME}-control-plane:/kind/bin/kubectl ${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/.kubectl
     chmod u+x ${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/.kubectl
     
-    _kubectl create -f $KIND_MANIFESTS_DIR/kube-flannel.yaml
```

Comment:
kind is doing that for us ?

Reply:
flannel does not support ipv6, so it cant be used, as I wrote in the commit message, it'll currently use kindnetd which is a simple provider, with single NIC support (no multus etc)

Reply:
so yes, it is doing this for us, but in a limited way.
anyhow - flannel can't be used with ipv6

Reply:
I suppose you'd need to make this executed by the other kind-based providers, right? Otherwise this PR introduces a regression to them.

Reply:
I'm not sure how this works where should I move it into, so it wont affect the other(s)?

Reply:
based on the name of the file I suspect that it is sourced from other providers. I did not check. I suspect that whichever sourced it, now need to do another extra step.

Reply:
ok, I'll take a look and fix

Reply:
I removed flannel installation from kind-1.17

and yes, afaik calico support ipv6 (and multus)

Reply:
@yuvalk this si removing it from 1.14 and 1.17 kind providers since it's common between the two, we either check KUBEVIRT_PROVIDER fore remove flannel or wait for calico to be merge and integrate it.